test/zero_one/geni/rdd_test.clj

```diff
@@ -53,7 +53,7 @@
     (rdd/master) => "local[*]"
     (rdd/persistent-rdds) => map?
     (rdd/resources) => {}
-    (rdd/spark-home) => nil?
+    (rdd/spark-home) => any?
```

Comment:
Actually, I believe this should be checked against the real Spark home using `(System/getenv "SPARK_HOME")`. Apologies, I should've said on the original issue thread!

Reply:
Good point, that is much better. Thanks! Just fixed it.